src/components/commons/sider/sider-body.tsx

```diff
@@ -90,6 +94,13 @@ const SiderBody = () => {
 				}
 			} else {
 				push('/');
+				enqueueSnackbar("Vous n'avez pas à être la", {
+					variant: 'error',
+					anchorOrigin: {
+						vertical: 'top',
+						horizontal: 'center',
+					},
+				});
```

Comment:
the message is displayed when erasing the name of the realm from the search bar

Reply:
problem resolve :D